File: components_page/components/layout/width.py
import dash_bootstrap_components as dbc
import dash_html_components as html

row = html.Div(
    [
        dbc.Row(dbc.Col(html.Div("A single, half-width column"), width=6)),
        dbc.Row(
            dbc.Col(html.Div("An automatically sized column"), width="auto")
        ),
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns"), width=3),
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns"), width=3),
            ]
        ),
    ]
)

File: components_page/components/layout/width.jl
using DashBootstrapComponents

row = html_div([
    dbc_row(dbc_col(html_div("A single, half-width column"), width = 6)),
    dbc_row(dbc_col(html_div("An automatically sized column"), width = "auto")),
    dbc_row([
        dbc_col(html_div("One of three columns"), width = 3),
        dbc_col(html_div("One of three columns")),
        dbc_col(html_div("One of three columns"), width = 3),
    ]),
]);
